addons/website_forum/models/forum.py

```diff
@@ -126,6 +126,8 @@ def _get_default_faq(self):
     karma_editor = fields.Integer(string='Editor Features: image and links',
                                   default=30, oldname='karma_editor_link_files')
     karma_user_bio = fields.Integer(string='Display detailed user biography', default=750)
+    karma_post = fields.Integer(string='Ask questions without validation', default=100)
+    karma_moderate = fields.Integer(string='Moderate posts', default=1000)
```

Comment:
To be able to moderate, you need "karma_moderate" (1000). "karma_post" is just to be able to ask a question without waiting for a validation from a moderator.
But in my opinion, we could set a lower value for the "karma_flag", as a flag is just used to mark a post as "potentially offensive". A flagged post will then appear in a specific queue, sorted by number of flags, and managed by moderators. If the post is marked as offensive by a moderator, its new state is "offensive", and it is removed from the forum ("closed" for a question, hidden for an answer). If it is not offensive, its new state is "active" again, and the flags are removed.